docs/source/extension_support.rst

```diff
@@ -0,0 +1,45 @@
+Extension Support
+=============
+
+Bit Manipulation Extension
+--------------------------------------------------------
+Setup RISCV-GCC compiler toolchain and ISS simulator
+----------------------------------
+
+1.  - Install `riscv-gcc`_ toolchain
+    - Download the repo and add " --enable-commitlog" at the end of line 6 in
+      `build_file`_ as it's not enabled by default
+    - Follow the `steps`_ to install GCC and spike
+2.  Update environment variable RISCV_GCC to the RISC-V gcc executable
+    executable. (example: <install_dir>/bin/riscv64-unknown-elf-gcc)
+3.  Update environment variable RISCV_OBJCOPY to RISC-V objcopy executable
+    executable. (example: <install_dir>/bin/riscv64-unknown-elf-objcopy)
+4.  Update environment variable SPIKE_PATH to the directory of the spike binary
+5.  Update `riscv-ovpsim`_ to Nov 26, 2019 or later version
+
+.. _steps: https://github.com/riscv/riscv-bitmanip/tree/master/tools#building-tools-with-draft-b-extension-instructions-support
+.. _riscv-gcc: https://github.com/riscv/riscv-bitmanip
+.. _build_file: https://github.com/riscv/riscv-bitmanip/blob/master/tools/riscv-isa-sim-build.sh
+.. _riscv-ovpsim: https://github.com/riscv/riscv-ovpsim
+
+Sample .bashrc setup::
+
+    export RISCV_TOOLCHAIN=<riscv_gcc_install_path>
+    export RISCV_GCC="$RISCV_TOOLCHAIN/bin/riscv64-unknown-elf-gcc"
+    export RISCV_OBJCOPY="$RISCV_TOOLCHAIN/bin/riscv64-unknown-elf-objcopy"
+    export SPIKE_PATH="$RISCV_TOOLCHAIN/bin"
+
+Run bitmanip simulation
```

Comment:
can you add a section to explain how to collect functional coverage for B-ext?

Reply:
Added, thanks